lib/watir/elements/date_field.rb

```diff
@@ -0,0 +1,39 @@
+module Watir
+  class DateField < Input
+
+    #
+    # Selects this radio button.
+    #
+
+    def set!(date)
+      message = "DateField##{__method__} only accepts instances of Date"
```

Comment:
Could we also support:
* Strings in the format "%Y-%m-%d" and
* Time objects?

In my application's date control, depending on what I am doing, I will use either the String format or a Date object - ie sometimes it's more convenient to directly specify the `String`.  I also use `active_support/time` to do things like `10.days.ago`. As this creates a `Time` object, it will be rejected by this validation.

Reply:
I just added code to parse Date or DateTime if it is a String. Huh, I didn't realize Rails did their days using Time class... We can support it the way that DateTmeField is supported, or the user can `#strftime` or `#to_date`

Reply:
Yeah, ok, I added support for Time class to both then.